kore/test/Test/Kore/Simplify/Or.hs

```diff
@@ -223,7 +223,7 @@ becomes
             ( stateIntention
                 [ prettyOr or1 or2
                 , "to become:"
-                , Unparser.unparse $ OrPattern.toTermLike expected
+                , Unparser.unparse $ OrPattern.toTermLike (mkSortVariable "_") expected
```

Comment:
If we have a sort here, we should use that. I guess we should use a sort variable only if the `OrPattern` is `\\bottom`.